src/main/java/de/novatec/marioai/tools/MarioAiRunner.java

```diff
@@ -88,7 +88,7 @@ public MarioAiRunner(MarioAiRunnerBuilder marioAiRunnerBuilder) {
 	
 	/**
 	 * 
-	 * @return List<EvaluationInfo> runs all agents using the current settings(variables) and returns the results
+	 * @return List of EvaluationInfo runs all agents using the current settings(variables) and returns the results
```

Comment:
`runs all agents using the current settings(variables)` sollte nach oben.
Entsprechend:
`@return List of EvaluationInfo (= result of the runs)`